dockerfiles/init/modules/compose/templates/docker-compose.yml.erb

```diff
@@ -24,6 +24,8 @@ che:
     - '/etc/passwd:/etc/passwd:ro'
 <% end -%>
   ports:
+    - '32001:32001'
```

Comment:
i don't think this is related to debug,in codenvy we have this always available. for example some monitoring tools require jmx connection and you may monitor app in production mode